Snowflake.Data/Core/Session/SFSession.cs

```diff
@@ -29,6 +29,8 @@ public class SFSession
 
         private const string SF_AUTHORIZATION_SNOWFLAKE_FMT = "Snowflake Token=\"{0}\"";
 
+        private const int SF_DEFAULT_QUERY_CONTEXT_CACHE_SIZE = 5;

```

Comment:
I know that we introduced those standards recently, but consts should use PascalCasing, maybe we need a separate PR to rename those globally in entire project? wdyt?

Reply:
I think it's good to have a separate PR to apply the naming rule to entire project since having different coding style between existing code and new changes is inconvenient. My concern is that it could introduce regression by mistake. Maybe we could use some automation tools for that? I'm not expert of C# what's your suggestion? If we don't have a tool to do that automatically I prefer to apply the naming rule to new changes only and leave the existing code as what it is.

Reply:
you changed it to _defaultQueryContextCacheSize. According to our convention it should be DefaultQueryContextCacheSize.

Reply:
https://github.com/snowflakedb/snowflake-connector-net/pull/757#discussion_r1309938836
A bit confusing what's the naming rule for private const.